custom_components/miele/translations/sensor.nb.json

```diff
@@ -23,7 +23,8 @@
       "normal_operation_mode": "Normalmodus",
       "own_program": "Eget program",
       "automatic_program": "Automatiskt program",
-      "cleaning_care_program": "Rengjøringsprogram"
+      "cleaning_care_program": "Rengjøringsprogram",
+      "maintainance_program": "Maintainance program"
```

Comment:
Spelling should be `maintenance` - in many places and files.

Reply:
wooops sorry :)